### DISCUSSION TITLE
0.14 release?

### DISCUSSION THREAD

**Discussion by raybellwaves** [anitagraser/movingpandas#272] [General]
Hi @anitagraser, could you possible cut a release.

If you are happy to share your method I can do the next one.

Release notes
--------------

New features:

https://github.com/anitagraser/movingpandas/pull/268 New add_angular_difference method

Includes fixes and enhancements for:

https://github.com/anitagraser/movingpandas/pull/267 Note that direction values are [0, 360) in documentation

https://github.com/anitagraser/movingpandas/pull/269 Fixed python dependencies

https://github.com/anitagraser/movingpandas/pull/269 Fixed read the docs build

**Comment by anitagraser** [comment]
Hi @raybellwaves, feel free to give it a try. Let me know if any of the steps is unclear: 

1. Update version in 
    1. `setup.py`
    1. `__init__.py `
1. Commit changes and create a **Github release/tag** for new version --> a Github action automatically creates the package and uploads it to PyPI, another action publishes the release to Zotero
1. Get hash from https://pypi.org/project/movingpandas/0.XXrc1
1. In **conda-forge** repo fork, e.g. [https://github.com/anitagraser/movingpandas-feedstock](https://github.com/anitagraser/movingpandas-feedstock?organization=anitagraser&organization=anitagraser)
    1. Update version and sha256 in `recipe/meta.yml`
    2. Commit 
    3. Open PR and follow instructions
    4. Check availability https://anaconda.org/conda-forge/movingpandas 
1. Update `changelog.md` in movingpandas **ghpages branch**
1. Add new version to **readthedocs**  https://readthedocs.org/projects/movingpandas/versions/

**Reply by raybellwaves** [reply]
@anitagraser can you give me admin access to https://readthedocs.org/projects/movingpandas please?

**Reply by anitagraser** [reply]
An invitation should be on its way

**Reply by raybellwaves** [reply]
I don't think I got an invite. Mind trying again?

**Reply by anitagraser** [reply]
@raybellwaves Yeah the invitation was market as expired. What's your rtd user name again?

**Reply by anitagraser** [reply]
I sent to raybellwaves
![image](https://user-images.githubusercontent.com/590385/211368180-6794f230-0b46-45b0-a1e0-49431be1acf1.png)

**Comment by anitagraser** [comment]
The release page should include the changelog https://github.com/anitagraser/movingpandas/releases

It is also copied to the archive at https://zenodo.org/record/7460290

**Comment by anitagraser** [comment]
The link to 0.14 on https://anitagraser.github.io/movingpandas/changelog is broken. I guess this was a manually set link? I'll go and fix it.

**Comment by michael-jacob** [comment]
@anitagraser Is the documentation page working as intended? Pages like https://movingpandas.readthedocs.io/en/main/trajectorycollection.html don't appear to be working for me. Thanks!

**Reply by anitagraser** [reply]
Hi Michael. Good catch! The main docs seem to be broken. The v0.12 release still works: 
https://movingpandas.readthedocs.io/en/v0.12/trajectorycollection.html

**Reply by michael-jacob** [reply]
Great-  I'll use v.012 release - thanks!
